apt/private/deb_translate_lock.bzl

```diff
@@ -42,13 +119,20 @@ def _deb_translate_lock_impl(rctx):
         if len(lockf.packages()) < 1:
             package_defs.append("   pass")
 
+    # TODO: rework lockfile to include architecure information
+    architectures = sets.make()
+    packages = sets.make()
```

Comment:
This re-work is what I did in #95 and later added in #100 the `get_architecture` for all this part, see:

* https://github.com/jjmaestro/rules_distroless/blob/b0f06d770cb6702179aa7fc994d1d1541cb4e8f7/apt/private/deb_translate_lock.bzl#L36
* https://github.com/jjmaestro/rules_distroless/blob/b0f06d770cb6702179aa7fc994d1d1541cb4e8f7/apt/private/lockfile.bzl#L47-L55

Reply:
Yeah, i'd be happy land that, but for now, i'll just workaround this.